api/types/types.go

```diff
@@ -388,6 +388,22 @@ type MountPoint struct {
 	Propagation mount.Propagation
 }
 
+// Task carries the information about one backend task
+type Task struct {
```

Comment:
Can you pls move these network specific Task and Service info to `api/types/network/network.go` ?